src/content/config.ts

```diff
@@ -54,12 +55,70 @@ const keynoters = defineCollection({
     }),
 });
 
+// Cache for fetched data to prevent duplicate network requests
+let cachedSpeakersData: any = null;
+let cachedSessionsData: any = null;
+
+// Shared data fetching function
+async function getCollectionsData() {
+  // Only fetch if not already cached
+  if (!cachedSpeakersData || !cachedSessionsData) {
+    const [speakersResponse, sessionsResponse] = await Promise.all([
+      fetch(
```

Comment:
just to clarify: how often does this cache actually update? I don’t see any TTL or similar mechanism in place here.

I know we’re syncing data between Pretalx and the program API every 10 minutes, but I’m not sure when those changes make their way to the frontend. Since the fetch only runs when the data isn’t cached, what triggers the cache to be reset? I know that now we use static data, but this PR should be merged once the API is live.